modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -131,3 +74,21 @@ Two plugins previously under the @janus-idp scope have moved to @red-hat-develop
 | `@red-hat-developer-hub/backstage-plugin-bulk-import-backend`
 |===
 
+As the scope of the previous plugins has been updated, the dynamic plugin configuration has also changed.
+
+[cols=2,%header]
+|===
+|*RHDH 1.3 Configuration*|*RHDH 1.4 Configuration*
+|link:https://github.com/janus-idp/backstage-showcase/blob/release-1.3/dynamic-plugins.default.yaml[dynamic-plugins.default.yaml]|link:https://github.com/janus-idp/backstage-showcase/blob/release-1.4/dynamic-plugins.default.yaml[dynamic-plugins.default.yaml]
+|===
+
+.Procedure
+* If you use any of the plugins listed above, you must revise your configuration to use the latest plugins from the new scope in order to upgrade from {product-very-short} 1.3 to {product-very-short} 1.4.
+
+[NOTE]
+====
+In addition to the tables above, you can also review which dynamic plugins have changed between {product-very-short} 1.3 and {product-very-short} 1.4 by comparing these two files:
```

Comment:
As per IBMSG, instead of using above we can use previous or preceding:
![image](https://github.com/user-attachments/assets/6ecdcebf-1e11-441f-8e5e-c11b276df900)

```suggestion
Along with the tables provided previously, you can compare the following files to see which dynamic plugins have changed between {product-very-short} 1.3 and {product-very-short} 1.4:
```